### ISSUE TITLE
Unable to resize window on wayland when native titlebar used

### ISSUE THREAD

**Issue by sempasha** [microsoft/vscode#235751]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.95.3 (Commit: f1a4fb101478ce6ec82fe9627c43efbf9e98c813)
- OS Version: Linux x64 6.11.10-200.fc40.x86_64 (Fedora Workstation 41)

Also tested on insiders build

- VS Code Version: 1.96.0-insider (Commit: 475acb4b9bdd77d4204224990bf288ed5bfe6b0e)
- OS Version: the same

Steps to Reproduce:

1. Change TitleBar style to `native`;
2. Start vs code with command `code --ozone-platform=wayland`. With or without options `--ozone-platform-hint` and `--enable-features` it does not matter;
3. Try to resize window with mouse;
4. You don't see any changes in coursro appearance when pointer close to application window edge or right above it.

The only ways to change window size are

1. Double click on titlebar to maximize window;
2. By calling `window.resizeTo(w, h);` from developer tools.

**Comment by sempasha** [comment]
#210624

**Comment by cbrnr** [comment]
I don't think that this issue is a duplicate, because it is not resolved (but is actually only present) if `--ozone-platform=wayland` is passed.

The problem is that due to an upstream Electron bug, window resizing currently does not work on Wayland, see https://github.com/electron/electron/issues/44543. Since Wayland is the default on many Linux distributions, this means that many people will run (or are currently running) into this issue.

It would be great if this issue could be re-opened until the problem is resolved.
